### PR TITLE
fix Mothur easyblock to work with 1.44.0 and higher

### PR DESCRIPTION
**Pull request by pescobar**
in newer mothur versions uchime should be included as a dependency

### CODE REVIEW

File: easybuild/easyblocks/m/mothur.py [hunk 1]
```diff
@@ -30,10 +30,11 @@
 import glob
 import os
 import shutil
+from distutils.version import LooseVersion
 
 from easybuild.easyblocks.generic.configuremake import ConfigureMake
 from easybuild.tools.build_log import EasyBuildError
-from easybuild.tools.modules import get_software_root
+from easybuild.tools.modules import get_software_root, get_software_version
```
[the remark this answers is not in the document]
Comment (pescobar) [reply easybuild/easyblocks/m/mothur.py:37]: isn't this import needed for `self.version` ?

[the remark this answers is not in the document]
Comment (pescobar) [reply easybuild/easyblocks/m/mothur.py:37]: @boegel ???

[the remark this answers is not in the document]
Comment (lexming) [reply easybuild/easyblocks/m/mothur.py:37]: @pescobar `self.version` is an attribute of the EasyBlock class. You can remove `get_software_version` in this case.

[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyblocks/m/mothur.py:37]: @pescobar ping

**Review by boegel — APPROVED** [review]
lgtm, thanks @pescobar !

**Pull request merged**